modules/integration/tests-integration/tests/src/test/java/org/wso2/carbon/integration/test/admin/EventProcessorAdminServiceTestCase.java

```diff
@@ -0,0 +1,196 @@
+            Assert.assertEquals(eventStreamManagerAdminServiceClient.getEventStreamCount(), --eventStreamCount);
+            eventProcessorAdminServiceClient.removeInactiveExecutionPlan("testPlan.siddhiql");
+            Assert.assertEquals(eventProcessorAdminServiceClient.getExecutionPlanConfigurationCount(), --executionPlanCount);
+
+        } catch (Throwable e) {
+            log.error("Exception thrown: " + e.getMessage(), e);
+            Assert.fail("Exception: " + e.getMessage());
+        } finally {
+            eventStreamCount = 0;
+            executionPlanCount = 0;
+        }
+    }
+
+    @Test(groups = {"wso2.cep"}, description = "Testing get siddhi streams and set statistics/tracing enable")
+    public void testGetSiddhiStreamsAndSetStatisticsAndTracingEnable() throws Exception {
+        String samplePath = "admin" + File.separator + "EventProcessorAdminServiceTestCase";
+
+        try {
+            eventStreamCount = eventStreamManagerAdminServiceClient.getEventStreamCount();
+            executionPlanCount = eventProcessorAdminServiceClient.getExecutionPlanConfigurationCount();
+
+            //Add input stream definition
+            String pizzaStreamDefinition = getJSONArtifactConfiguration(samplePath, "org.wso2.sample.pizza.order_1.0.0.json");
+            eventStreamManagerAdminServiceClient.addEventStreamAsString(pizzaStreamDefinition);
+            Assert.assertEquals(eventStreamManagerAdminServiceClient.getEventStreamCount(), ++eventStreamCount);
+
+            //Add output stream definition
+            String outStreamDefinition = getJSONArtifactConfiguration(samplePath, "outStream_1.0.0.json");
+            eventStreamManagerAdminServiceClient.addEventStreamAsString(outStreamDefinition);
+            Assert.assertEquals(eventStreamManagerAdminServiceClient.getEventStreamCount(), ++eventStreamCount);
+
+            //Add execution plan
+            String executionPlan = getExecutionPlanFromFile(samplePath, "testPlan.siddhiql");
+            eventProcessorAdminServiceClient.addExecutionPlan(executionPlan);
+            Assert.assertEquals(eventProcessorAdminServiceClient.getExecutionPlanConfigurationCount(), ++executionPlanCount);
+
+            //Set statistics and tracing enable
+            eventProcessorAdminServiceClient.setStatisticsEnabled("testPlan", true);
+            eventProcessorAdminServiceClient.setTracingEnabled("testPlan", true);
+
+            //Get active execution plans
+            ExecutionPlanConfigurationDto executionPlanConfigurationDto = eventProcessorAdminServiceClient.getActiveExecutionPlanConfiguration("testPlan");
+            Assert.assertEquals(executionPlanConfigurationDto.getName(), "testPlan");
+
+            //Get siddhi stream count
+            StreamDefinitionDto[] streamDefinitionDtos = eventProcessorAdminServiceClient.getSiddhiStreams(executionPlan);
+            Assert.assertEquals(streamDefinitionDtos.length, eventStreamCount);
```

Comment:
@thiliA 
Will this assertion hold if the server already has streams deployed from previous tests that are not part of this particular execution plan? e.g. total 4 streams are used in server, but only 2 are used by this execution plan. In that case, I think this test case would break. Can you fix it so that the test case wouldn't break in such a scenario?